src/Console/Command/ObfuscateData.php

```diff
@@ -0,0 +1,225 @@
+<?php namespace Ushahidi\Console\Command;
+
+/**
+ * Ushahidi Obfuscate Data console command
+ *  - This overwrites specific data within the current database
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Illuminate\Console\Command;
+use Ushahidi\Core\Entity\PostRepository;
+use Ushahidi\Core\Entity\ContactRepository;
+use Ushahidi\Core\Entity\ConfigRepository;
+use Ushahidi\Core\Entity\UserRepository;
+use Ohanzee\DB;
+use Faker;
+
+class ObfuscateData extends Command
+{
+    private $postRepository;
+    private $contactRepository;
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'db:obfuscatedata';
+
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'db:obfuscatedata';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Obfuscates selected PII and removes various config keys from database';
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function handle()
+    {
+        $this->db = service('kohana.db');
+        $this->contactRepository = service('repository.contact');
+        $this->postRepository = service('repository.post');
+        $this->configRepository = service('repository.config');
+        $this->userRepository = service('repository.user');
+
+        if ($this->isTestDeployment() || $this->isStagingDeployment()) {
+            //confirm acknowledgement
+            $this->info("This script will wipe data from the current deployment.\n\n");
+
+            if (!$this->confirm('Do you acknowledge that this data will still contain sensitive data?')) {
+                $this->info("Request canceled.\n");
+                exit;
+            }
+
+            // Do overwriting
+            $this->db->begin();
+            $this->overwriteContacts();
+            $this->overwritePostAuthors();
+            $this->removeDataProviderValues();
+            $this->overwriteSiteConfig();
+            $this->deleteUsers();
+            $this->addAdminUser();
+            $this->db->commit();
+        } else {
+            $this->info("This script will only run on test or staging deployments.\n\n");
+            exit;
+        }
+    }
+
+    private function isTestDeployment()
+    {
+        return (strtolower(getenv('APP_ENV')) == 'test');
```

Comment:
You should use app()->environment for these checks

Reply:
Maybe add a static property of `allow_environments` and check against that?

Reply:
@rjmackay - Not sure if app() is available from within a Console command, but I can access with `$this->getLaravel()->environment();`
Am I doing that wrong?

Reply:
I think `app()` is available everywhere as a helper function. I don't know if `this->app` works in commands, but getLaravel() seems reasonable. 
Otherwise check the docs here: https://lumen.laravel.com/docs/5.1#custom-configuration-files